<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>printdemoClass</class>
 <widget class="QMainWindow" name="printdemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>printdemo</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="printAction"/>
    <addaction name="exportAction"/>
    <addaction name="selectFontAction"/>
    <addaction name="printChineseAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="printAction">
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="exportAction">
   <property name="text">
    <string>导出PDF</string>
   </property>
  </action>
  <action name="selectFontAction">
   <property name="text">
    <string>选择字体</string>
   </property>
  </action>
  <action name="printChineseAction">
   <property name="text">
    <string>打印中文</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="printdemo.qrc"/>
 </resources>
 <connections/>
</ui>
